src/lib/server-utilities.js

```diff
@@ -0,0 +1,9 @@
+import fs from 'fs';
+import crypto from 'crypto';
+
+export function md5Hash(file) {
+  const hashInstance = crypto.createHash('md5');
+  // readFileSync in the syncronous version of readFile
+  const fileContents = fs.readFileSync(file, 'utf8');
+  return hashInstance.update(fileContents).digest('hex');
+}
```

Comment:
The naming here isn't very consistent. Either `falcorUtils` should become `falcor-utilities` or this (and the other `x-utilities`) should become `serverUtils` (or `xUtils`). To be honest, I'm neutral on which we use, but I think it's more clear to use a single file naming standard.

The Airbnb style guide isn't very clear on this. They suggest using Pascal case for filenames in the React style guide, but I assume that's more for regular React components. In the JavaScript style guide, there are examples of all different kinds of file naming conventions that seem to be fine.